schema-admin.graphql

```diff
@@ -674,6 +765,14 @@ type Mutation {
         data: CreateApprovedCuratedCorpusItemInput!
     ): ApprovedCuratedCorpusItem!
 
+    """
+    Lets an automated process create an Approved Item and optionally schedule it to appear
+    on a Scheduled Surface.
+    """
+    loadApprovedCuratedCorpusItem(
```

Comment:
i don't love "load" - it could be interpreted as "load from this system" as opposed to "load from another system". what about "import"?

Reply:
I love `import`, @jpetto , thank you for this suggestion! This also means the terribly named `LoadApprovedCuratedItemPayload` can become `ImportApprovedCuratedItemPayload` instead :).